packages/trader/src/Modules/MT5/Components/mt5-account-card.jsx

```diff
@@ -36,51 +38,64 @@ const MT5AccountCard = ({
     onPasswordManager,
 }) => {
     const IconComponent = icon || (() => null);
-    const lbl_add_account =
-        type.category === 'real' ? (
-            <Localize i18n_default_text='Add real account' />
-        ) : (
-            <Localize i18n_default_text='Add demo account' />
-        );
+    const BackgroundComponent = background_image || (() => null);
+    const lbl_add_account = <Localize i18n_default_text='Add account' />;
     const cta_label = button_label || lbl_add_account;
-
-    const has_popular_banner = type.type === 'synthetic' && type.category === 'real';
     const has_demo_banner = type.category === 'demo';
 
-    return (
-        <div className={classNames('mt5-account-card', { 'mt5-account-card__logged-out': !is_logged_in })}>
-            {has_popular_banner && (
-                <div className='mt5-account-card__banner'>
-                    <Localize i18n_default_text='Most popular' />
-                </div>
-            )}
-            {has_demo_banner && (
-                <div className='mt5-account-card__banner mt5-account-card__banner--demo'>
-                    <Localize i18n_default_text='DEMO' />
+    const renderHeader = () => {
+        return (
+            <>
+                <div
+                    className={classNames('mt5-account-card__type', {
+                        'mt5-account-card__type': has_demo_banner,
+                    })}
+                    id={`mt5_${type.category}_${type.type}`}
+                >
+                    <div className='mt5-account-card__type--description'>
+                        <h1 className='mt5-account-card--paragraph mt5-account-card--paragraph-bold'>{title}</h1>
+                        {has_demo_banner && (
+                            <div className='mt5-account-card__banner mt5-account-card__banner--demo'>
+                                <Localize i18n_default_text='Demo' />
+                            </div>
+                        )}
+                        {existing_data && existing_data.display_balance && is_logged_in && (
```

Comment:
```suggestion
                        {existing_data?.display_balance && is_logged_in && (
```

Reply:
Code has been done by others but i will update it.